File: Hue.Web/src/model/artist/Service.ts
import Identifiable from "../Identifiable";
import Artist from "./Artist";
import ServiceAddition from "./ServiceAddition";

export default class Service extends Identifiable {
    public name: string = "";
    public description: string = "";
    public basePrice: number = 0;
    public currency: string = "USD";
    public additions: ServiceAddition[] = []
    public artist: Artist = {} as Artist
    public commissionType: number = 0

}

File: Hue.Common/Artist/Service.cs
using Hue.Common.Commission;

namespace Hue.Common.Artist {
    public class Service : Identifiable {

        
        public CommissionType CommissionType { get; set; }

        public Artist Artist { get; set; } = new();

        public string Name { get; set; } = "";
        public string Description { get; set; } = "";
        public int BasePrice { get; set; } = 0;
        public string Currency { get; set; } = "";

        public List<ServiceAddition> Additions { get; set; } = [];

    }
}
